teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', _parent_permission: true)
+    super(age, name)
+    @specialization = specialization
```

Comment:
- [ ] parent_permission should be included as an argument when calling super as well.